_opensearch/install/rpm.md

```diff
@@ -5,154 +5,472 @@ parent: Install OpenSearch
 
-2. Import the public GPG key. This key verifies that the your OpenSearch instance is signed.
+This guide assumes that you are comfortable working from the Linux command line interface (CLI). You should understand how to input commands, navigate between directories, and edit text files. Some example commands reference the `vi` text editor, but you may use any text editor available.
+{:.note}
 
-   ```bash
-   sudo rpm --import https://artifacts.opensearch.org/publickeys/opensearch.pgp
-   ```
+## Download and install OpenSearch
 
-3. On your host, use `sudo yum install` or `sudo rpm -ivh` to install the package. 
+### Install OpenSearch from a package
 
+1. Download the RPM package for the desired version directly from the [OpenSearch downloads page](https://opensearch.org/downloads.html){:target='\_blank'}. The RPM package can be download for both **x64** and **arm64** architectures.
+1. Import the public GPG key. This key verifies that your OpenSearch instance is signed.
+    ```bash
+    sudo rpm --import https://artifacts.opensearch.org/publickeys/opensearch.pgp
+    ```
+1. From the command line interface (CLI), you can install the package with `rpm` or `yum`.
    **x64**
-
    ```bash
+   # Install the x64 package using yum.
    sudo yum install opensearch-{{site.opensearch_version}}-linux-x64.rpm
-   sudo yum install opensearch-dashboards-{{site.opensearch_version}}-linux-x64.rpm
-   ```
-
-   ```bash
+   # Install the x64 package using rpm.
    sudo rpm -ivh opensearch-{{site.opensearch_version}}-linux-x64.rpm
-   sudo rpm -ivh opensearch-dashboards-{{site.opensearch_version}}-linux-x64.rpm
    ```
-  
    **arm64**
-
    ```bash
+   # Install the arm64 package using yum.
    sudo yum install opensearch-{{site.opensearch_version}}-linux-x64.rpm
-   sudo yum install opensearch-dashboards-{{site.opensearch_version}}-linux-arm64.rpm
-   ```
-
-   ```bash
+   # Install the arm64 package using rpm.
    sudo rpm -ivh opensearch-{{site.opensearch_version}}-linux-x64.rpm
-   sudo rpm -ivh opensearch-dashboards-{{site.opensearch_version}}-linux-arm64.rpm
    ```
-  
-   Once complete, you can run OpenSearch inside your distribution.
-
-## YUM method
-
-YUM, an RPM package management tool, allows you to pull the RPM package from the YUM repository library. 
-
-1. Create a repository file for both OpenSearch and OpenSearch Dashboards:
-
+1. After the installation succeeds, enable OpenSearch as a service.
+    ```bash
+    sudo systemctl enable opensearch
+    ```
+1. Start OpenSearch.
+    ```bash
+    sudo systemctl start opensearch
+    ```
+1. Verify that OpenSearch launched correctly.
+    ```bash
+    sudo systemctl status opensearch
+    ```
+
+### Install OpenSearch from a local YUM repository
+
+YUM, the primary package management tool for Red Hat-based operating systems, allows you to download and install the RPM package from the YUM repository. 
```

Comment:
maybe add link to YUM RPM instructions

[YUM]({{site.url}}{{site.baseurl}}/opensearch/install/rpm/#yum-method